easybuild/easyconfigs/b/Boost/Boost-1.67.0-CrayGNU-19.08.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'boostcray'
```

Comment:
I think we should deprecate this `boostcray` easyblock. If this builds work with the standard easyblock from EB.
(se PR #1212), it could be safely removed.

Reply:
Done.

Reply:
It failed when I removed `easyblock = 'boostcray'`. So, I added it again.